spec/services/api/v3/participant_declarations_query_spec.rb

```diff
@@ -19,15 +19,14 @@
   let(:school_cohort3) { create(:school_cohort, :fip, :with_induction_programme, lead_provider: lead_provider2, cohort: cohort2) }
   let(:participant_profile4) { create(:ect, :eligible_for_funding, school_cohort: school_cohort3, lead_provider: lead_provider1) }
 
+  let(:npq_only_lead_provider) { create(:cpd_lead_provider, :with_npq_lead_provider) }
```

Comment:
Thinking about this, is it worth removing NPQ completely from this spec 🤔 unless we keep it until we clean up the data? what do you think